samples/Directory.Build.targets

```diff
@@ -299,5 +299,6 @@
     <PackageReference Update="Xamarin.GooglePlayServices.Measurement.Base" Version="121.1.0" />
     <PackageReference Update="Xamarin.GooglePlayServices.Measurement.Api" Version="121.1.0" />
     <PackageReference Update="Xamarin.CheckerFramework.CheckerQual" Version="3.25.0" />
+    <PackageReference Update="Xamarin.Build.Download" Version="0.11.4" />
```

Comment:
I think correct filename for Central package management should be `Directory.packages.targets` or even better `Directory.packages.props`

Reply:
This just uses the existing package versioning solution we have, it does not attempt to migrate to a new one. We can do that in the future if we want.